backend/s3/s3.go

```diff
@@ -655,14 +655,30 @@ isn't set then "acl" is used instead.`,
 		}, {
 			Name:     "sse_kms_key_id",
 			Help:     "If using KMS ID you must provide the ARN of Key.",
-			Provider: "AWS",
+			Provider: "AWS,Ceph,Minio",
 			Examples: []fs.OptionExample{{
 				Value: "",
 				Help:  "None",
 			}, {
 				Value: "arn:aws:kms:us-east-1:*",
 				Help:  "arn:aws:kms:*",
 			}},
+		}, {
```

Comment:
I wonder if these (and the other encryption options) should be in the Advanced config section?

Reply:
I've put the new options into advanced config, but left the existing ones,